.github/workflows/colcon.yaml

```diff
@@ -0,0 +1,49 @@
+name: Colcon Build Test
+on:
+  push:
+    branches:
+    - 'rolling'
+    - 'iron'
+    - 'humble'
+  pull_request:
+    branches:
+    - '*'
+jobs:
+  build-test:
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {rosdistro: 'rolling', container: 'ros:rolling'}
```

Comment:
Can we use rolling when the target branch is not rolling?

Reply:
I'll manually change this on the backport.